src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -75,7 +75,7 @@ fun RecommendationEntity.toRecommendationResponse(): RecommendationResponse {
 data class RecommendationModel(
   val crn: String?,
   var sensitive: Boolean? = null,
-  var ppudRecordPresent: Boolean? = null,
+  var ppudRecordPresent: Boolean? = false,
```

Comment:
If we are initialising this to false, can we make it non nullable?

Reply:
Done 👍